backend/build_docs.py

```diff
@@ -21,6 +21,7 @@
     'https://raw.githubusercontent.com/The-OpenROAD-Project/OpenSTA/master/README.md'
 )
 yosys_html_url = 'https://yosyshq.readthedocs.io/projects/yosys/en/latest'
+klayout_html_url = 'https://www.klayout.de/doc.html'
```

Comment:
I know this is not versionable. Add a todo to upload a versioned copy (date) of all html files to huggingface repo

You can put all the data sources that cannot be versioned under this, and pull the huggingface data accordingly.